updated_main.py

```diff
@@ -3,57 +3,86 @@
 import bs4
 import time
 import json
+import os
+from email.message import EmailMessage
+import ssl
+import smtplib
 
 # Running a for loop to get the latest price of the GBP
 for i in range(100):
 
     # Now extracting the value of British Pound in Indian Rupees
-    currency = "convert 1 pound to rs"
+    def Getting_current_price():
+        currency = "convert 1 pound to rs"
+        url = "https://google.com/search?q=" + currency
```

Comment:
```suggestion
        url = f"https://google.com/search?q={currency}"
```
Use f-strings everywhere.

Reply:
See: https://realpython.com/python-f-strings/#:~:text=Also%20called%20%E2%80%9Cformatted%20string%20literals,the%20__format__%20protocol..

In particular read section on speed.